### PR TITLE
Create : Home Page created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Create : home created

Screenshot_2022-02-18_13-11-31

preview link : https://deploy-preview-2--fabelle-learn.netlify.app/

### CODE REVIEW

**Review by Developers-codz — CHANGES_REQUESTED** [review]
UI is Looking good to me, just some observations
1. In card, there must be add to cart button not go to card
2. also button size is too large for cards
3. In footer you can provide a heading for each category and it would be better to put social media link  horizontal
4. The below area is looking very blank can 
abcde
increase font size I guess.

File: index.html [hunk 1]
```diff
@@ -1,12 +1,279 @@
...
+
+        <!-- Top Categories -->
+        <section class="top-categories">
+            <h2>Top Categories</h2>
+            <div class="categories-container">
+                <!-- cards -->
+                <a href="#" class="card-container">
+                    <div class="img-wrapper">
+                        <img src="./images/homepage/development.svg" alt="">
+                    </div>
```
Comment (vrinda-mahajan) [inline index.html]: Provide value to the alt attribute.

**Review by vrinda-mahajan — APPROVED** [review]
Just, a few suggestions on the UI.
Decrease the size of nav icon text and provide it with a lighter font weight.
Also, increase the size of the content on the landing page.

Screenshot (67)

File: index.html (hunk 1 above)
Comment (arjun-computer-geek) [reply index.html]: Ok i will do it

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    justify-content: center;
+}
+
+main section .represent-img {
+    width: 50%;
+    height: 100vh;
+}
+
+main section .represent-img img {
+    width: 100%;
```
Comment (MayankKumar10) [inline css/style.css:55]: make it an image helper class

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+main section h2 {
+    border-bottom: 2px solid black;
+}
+
+main .popular-courses .courses-container {
+    height: 80vh;
+    width: 100%;
+    display: flex;
```
Comment (MayankKumar10) [inline css/style.css:71]: make flex a helper class so you don't have to repeat it.

File: index.html [hunk 4]
```diff
@@ -1,12 +1,279 @@
...
+            <a class="nav-link" href="">Categories</a>
+        </div>
+        <div class="nav-search">
+            <i class="fa fa-search"></i>
+            <input type="text" placeholder="Type to search">
+        </div>
+        <div class="nav-icons">
+            <a href="">
+                <i class="fa fa-user"></i>
+                <span class="nav-icon-text">Login</span>
```
Comment (MayankKumar10) [inline index.html:39]: just add some padding in Login, wishlist, and cart

**Review by MayankKumar10 — APPROVED** [review]
Everything else Looks good to me you can merge it.

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, just one suggestion in the ui, it would look good if you can add some hover properties on login, Wishlist and cards

**Pull request merged**